backend/pkg/kafka/connection_helper.go

```diff
@@ -23,8 +23,8 @@ func NewSaramaConfig(cfg *Config) (*sarama.Config, error) {
 	}
 	sConfig.ClientID = cfg.ClientID
 	sConfig.Version = version
-	sConfig.Net.KeepAlive = 30 * time.Second
-	sConfig.Net.DialTimeout = 10 * time.Second
+	sConfig.Net.KeepAlive = 15 * time.Second
```

Comment:
KeepAlive will eventually end up here: https://godoc.org/net#Dialer.KeepAlive
Does it make sense to have the same value for KeepAlive as we have for Read/Write Timeout?
Shouldn't we send KeepAlive more often? Or am I misinterpreting how it works?

Reply:
The KeepAlive is used for the TCP connection while the Dial, Read and Write timeout are not relevant for keeping the TCP connection alive, but they are used in Sarama for timeouts. Hence they can be all the same. I lowered them to 15s because sarama will wait up to ReadTimeout if a broker is dead (and the cached metadata is not yet updated).